### PR TITLE
Implement `assignee` query field

### PR DESCRIPTION
**Pull request by eupn**

### CODE REVIEW

File: src/search/query.rs [hunk 1]
```diff
@@ -55,6 +56,14 @@ impl Query {
         self
     }
 
+    /// *Adds* an `assignee` statement to the query.
+    ///
+    /// Results in `assignee:username`.
+    pub fn assignee(mut self, assignee: &str) -> Self {
```
Comment (spenserblack) [inline src/search/query.rs]: Could you make the function parameters match the documentation? i.e. `pub fn assignee(mut self, username: &str)`. While unlikely, there's a chance that someone could interpret the value they pass for assignee to be *before* the `:`, not after.

**Review by spenserblack — CHANGES_REQUESTED** [review]
Thanks for the PR! Left a note.

File: src/search/query.rs (hunk 1 above)
Comment (eupn) [reply src/search/query.rs]: Yeah, let's do that.

**Review by spenserblack — APPROVED** [review]
Looks good! :+1: 
~~Unfortunately, #22 has caused a merge conflict. Could you resolve this conflict so I can merge your PR?~~

**Pull request merged**